config/deploy.rb

```diff
@@ -0,0 +1,13 @@
+lock '3.2.1'
+
+set :application, 'coursemology'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.